packages/node/src/coordinator/calls/aggregation.test.ts

```diff
@@ -24,7 +24,7 @@ describe('aggregate (API calls)', () => {
     const res = aggregation.aggregate(fixtures.buildConfig(), apiCalls);
     expect(res).toEqual({
       apiCallId: {
-        requesterIndex: '3',
+        sponsorAddress: '3', //TODO: fix value
```

Comment:
I don't really like leaving all of these TODOs in the codebase. What's preventing us from fixing this now?

Reply:
I know and I feel the same way but after merging this PR and this other one https://github.com/api3dao/airnode/pull/523 I'll be to compile and run the test. My next task is actually fixing all broken tests in the node package and I plan to remove all TODOs as my next task.

Reply:
Ok as long as they won't be left in the code for long 👍🏻

Reply:
I promise 😇